src/MentionsInputChildren.spec.tsx

```
@@ -213,21 +213,50 @@ describe('MentionsInputChildren', () => {
+      </>
+    ).config
+
+    expect(areMentionConfigsEqual(stickyConfig, nonStickyConfig)).toBe(true)
+  })
+
   it('prepares trigger query regex metadata once with normalized flags', () => {
     const [stringTriggerConfig, regexTriggerConfig] = prepareMentionsInputChildren(
       <>
         <Mention trigger="@" data={[]} />
-        <Mention trigger={/(#([\p{L}\d_]*))$/giu} data={[]} />
+        <Mention trigger={/(#([\p{L}\d_]*))$/giuy} data={[]} />
```

ESLint

[error] Remove the 'g' flag from this regex as it is shadowed by the 'y' flag.

(sonarjs/stateful-regex)